.github/ISSUE_TEMPLATE/new_release.md

```diff
@@ -6,37 +6,81 @@ labels: "kind/release"
 assignees: '@sayanchowdhury'
 ---
 
-## Release Flatcar Container Linux <Alpha-VERSION> [<Beta-VERSION>] [<Stable-VERSION>]
-  
-The release of Flatcar Container Linux <VERSION> [<VERSION> ...] is planned for <MONTH> <DAY>, <YEAR>. 
-
-### Tasks
-- [ ] Go / No Go [hackmd doc](https://hackmd.io/team/flatcar?nav=overview) created from previous one for test tracking and release notes: [link](TODO) ()
-- [ ] Check that the kernel PR are merged for the channel(s) to be released. ()
-- [ ] All planned changes are merged & blocker bugs are fixed, last nightly reviewed for problems ()
-- [ ] Branch off main (only new major) ()
-- [ ] Tag build branch ()
-- [ ] Start Jenkins builds, post links as comment ()
-- [ ] Test status captured in hackmd doc ()
-- [ ] All tests are green ()
-- [ ] Release notes / announcements added to Go/No Go doc from `container/image_changes` job by selecting `Timestamps: None` ()
-- [ ] QA: check diff for image file list, packages, and image size in the image job output ()
-- [ ] Matrix Go/No Go meeting (inc. release notes review) ()
-- [ ] Update payload signed/uploaded ()
-- [ ] `scripts` release created on GitHub ()
-- [ ] Release job `container/release` ran ()
-- [ ] Images uploaded with `copy-to-origin.sh` ()
-- [ ] Azure offer updated & publishing started ()
-- [ ] Azure Go Live ()
-- [ ] GCP Offer deployment package uploaded to Google bucket and submitted for verification ()
-- [ ] GCP Offer submit for verification ()
-- [ ] GCP Offer Publish ()
-- [ ] CAPI for new Stable:
+The release of the following Flatcar Container Linux version is planning <MONTH> <DAY>, <YEAR>.
+
+- Alpha <VERSION>
+- Beta <VERSION>
+- Stable <VERSION>
+- LTS <VERSION>
+
+## 1. Preparation
+
+Owner:
+
+- [ ] A. Go/No-Go hackmd doc created from HackMD template for test tracking and release notes: (update the link here)
```

Comment:
Maybe the https://hackmd.io/team/flatcar?nav=overview link is useful for those that have to create one and don't remember how

Reply:
Ah! yes. This link is in local, maybe I forgot to push. I will push the change.